CHANGELOG.md

```diff
@@ -14,10 +26,6 @@ These should not impact the functionality of the module.
 - Add OMAPI key configuration [\#90](https://github.com/voxpupuli/puppet-dhcp/issues/90)
 - Add define for specifying DHCP classes [\#86](https://github.com/voxpupuli/puppet-dhcp/issues/86)
 
-**Breaking changes:**
```

Comment:
Could you leave this in?

Reply:
oh :(
That got lost somehow

Reply:
regenerated the changelog. looks better now